modules/apps/29-fee/keeper/keeper_test.go

```diff
@@ -57,6 +63,14 @@ func SetupFeePath(path *ibctesting.Path) error {
 	return nil
 }
 
+func (suite *KeeperTestSuite) NewPacketId(id uint64) channeltypes.PacketId {
+	return channeltypes.PacketId{ChannelId: ibctesting.FirstChannelID, PortId: types.PortKey, Sequence: id}
+}
+
+func (suite *KeeperTestSuite) NewIdentifiedPacketFee(id channeltypes.PacketId, fee types.Fee) types.IdentifiedPacketFee {
+	return types.IdentifiedPacketFee{PacketId: &id, Fee: fee, Relayers: []string(nil)}
```

Comment:
This function should go in the `types` package similar to other helpers. see -> https://github.com/cosmos/ibc-go/blob/ics29-fee-middleware/modules/apps/29-fee/types/msgs.go#L17

Reply:
it's just a helper function for testing though, if we want to make it a full helper i would add the relayers array into the function params

Reply:
I think this should exist in `package types`, it's wrong for it to exist as a method on the KeeperTestSuite. 

Right now it's just a helper function for testing, but I think in your follow-up PR for calling the escrow logic funcs you could make use of it and then it wouldn't make sense to have it here - e.g. [`EscrowPacketFee`](https://github.com/cosmos/ibc-go/blob/ics29-fee-middleware/modules/apps/29-fee/keeper/escrow.go#L15) you may want to use `NewIdentifiedPacketFee()` to construct that type, and pass it as an arg

Reply:
https://github.com/cosmos/ibc-go/blob/issue%23479-grpc-query/modules/apps/29-fee/types/msgs.go#L165

I actually already added this on the previous PR. You can use this. Sorry, I just realized :)